autoscheduler/frontend/src/package.json

```diff
@@ -16,11 +16,12 @@
     "@testing-library/jest-dom": "^4.1.2",
     "@testing-library/react": "^9.3.0",
     "@types/classnames": "^2.2.9",
-    "@types/jest": "^24.0.19",
+    "@types/jest": "^24.0.23",
     "@types/react": "^16.9.9",
     "@types/react-dom": "^16.9.1",
     "@typescript-eslint/eslint-plugin": "^2.4.0",
     "@typescript-eslint/parser": "^2.4.0",
+    "css-loader": "^3.2.0",
```

Comment:
I changed the `typings-for-css-modules-loader` to use a fork by some guy who updated it to work with the latest CSS loader